tests/test_additional_responses_bad.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from fastapi import FastAPI
+from starlette.testclient import TestClient
+
+app = FastAPI()
+
+
+@app.get("/a", responses={"hello": {"description": "Not a valid additional response"}})
+async def a():
+    return "a"
```

Comment:
```suggestion
    pass  # pragma: no cover
```
I think this will make the test coverage pass; this is how it's done elsewhere where the goal is not to test the actual endpoint.

Reply:
i was in the middle of checking why the coverage dropped but haven't managed to reach any conclusion yet. thanks a lot, this comment saved a lot of time for me.